frontend/src/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
-import ReactDOM from 'react-dom'
+import { createRoot } from 'react-dom/client'
 import './index.css'
 import { App } from './App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+
+const root = createRoot(document.getElementById('root'))
+root.render(<App/>)
```

Comment:
Nice to see that you found a solution to the error you get now with ReactDom being outdated.